### ISSUE TITLE
Wrong entities mapping

### ISSUE THREAD

**Issue by Daddaniele** [PrestaShop/PrestaShop#21206]
There is an error on PrestaShopBundle\Entity\Tab.php 
The entity refers to TabLang entity with mapping: tab

![image](https://user-images.githubusercontent.com/53485203/94550694-71854580-0254-11eb-80aa-f40b44134517.png)

The mappedBy should be "id" as defined on PrestaShopBundle\Entity\TabLang.php 
![image](https://user-images.githubusercontent.com/53485203/94550739-88c43300-0254-11eb-9ec6-de3da9cf6e6b.png)

1. Go to configure/advanced/performance
2. Set debug on true
3. this will activate the symfony debug toolbar 
4. look for invalid entities alert

**Screenshots**

![image](https://user-images.githubusercontent.com/53485203/94550983-e789ac80-0254-11eb-8809-a10430b0f4c3.png)
![image](https://user-images.githubusercontent.com/53485203/94551008-f53f3200-0254-11eb-9115-869b5a77d316.png)

#### Additional information

* PrestaShop version: 1.7.6.7
* PHP version: tried with 7.0 and 7.3

**Comment by khouloudbelguith** [comment]
Hi @Daddaniele,

Thanks for these clarifications.
I manage to reproduce the issue with 1768, 177x branch and develop branch.
![image](https://user-images.githubusercontent.com/16067358/94552111-abefe200-0256-11eb-85dd-dd668c742cbe.png)
I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!

@Daddaniele, 
PrestaShop is not compatible with PHP 7.3, in fact, PrestaShop 1.7.0 to 1.7.3 is compatible with PHP 7.1 and PrestaShop 1.7.4 & 1.7.5 & PS1.7.6 are compatible with PHP 7.2.
For more details, you can follow this link: http://build.prestashop.com/news/prestashop-1-7-is-moving-to-symfony-3-4-and-php-5-6/
Next Major release PS1.7.7 will be compatible with PHP7.3.
You need to downgrade your PHP version.

Thanks!

**Comment by karpoj23** [comment]
I see that it is solved in develop branch... with this code:

```

 private $icon;

    /**
     * @ORM\OneToMany(targetEntity="PrestaShopBundle\Entity\TabLang", mappedBy="id")
     */
    private $tabLangs;
```

but to solved this problem in 1.7.6.7... do you have a solution? I have tried the change and remove cache but it is not working...
